x-pack/plugins/ml/common/types/saved_objects.ts

```diff
@@ -61,11 +61,11 @@ export interface SyncCheckResponse {
   result: boolean;
 }
 
-export interface DeleteJobCheckResponse {
-  [jobId: string]: DeleteJobPermission;
+export interface DeleteMLSpaceAwareItemCheckResponse {
```

Comment:
Nit - for consistency with `CanDeleteMLSpaceAwareItemsResponse` should it be plural `Items` in the interface name here?

Reply:
https://github.com/elastic/kibana/pull/127438/commits/fb5876397d67bc251532a53dd423ee31da1b33a0